src/lib/sort-rules.js

```diff
@@ -1,9 +1,27 @@
 'use strict'
 
 function getSortedRules(rules) {
-  return rules.sort(function sort(a, b) {
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(rules) {
+  var sortedRules = []
+
+  Object.keys(rules)
+    .sort(sortAlphabetically)
+    .forEach(function map(ruleName) {
+      sortedRules.push(ruleName, rules[ruleName].config1, rules[ruleName].config2)
```

Comment:
Am I reading this correct?

```js
// sortedRules
['foo-rule', '0 foo', '2 foo', 'bar-rule', '2 bar', '1 bar', ...]
```
If I am reading this correct, I am afraid, it would not be difficult to maintain. I would expect it to be

```js
[{ 'foo-rule': {config1: '0 foo', config2:  '2 foo'} }, ....]
```